narf/ioutils.py

```diff
@@ -318,21 +319,20 @@ def script_command_to_str(argv, parser_args):
             call_args[select] = np.vectorize(lambda x: f"'{x}'")(call_args[select])
     return " ".join([argv[0], *call_args])
 
-def make_meta_info_dict(exclude_diff='notebooks', args=None):
+def make_meta_info_dict(exclude_diff = 'notebooks', args = None, wd = pathlib.Path(__file__).parent):
```

Comment:
Is this actually the correct path?

What's actually desired here would be the top level directory of the repository, but this would point to the "narf" subdirectory

Might also be better to define this path in a common place.